graph/repo.go

```diff
@@ -59,3 +73,28 @@ func TryMakeURI(cloneURL string) (string, error) {
 func URIEqual(a, b string) bool {
 	return strings.EqualFold(a, b)
 }
+
+// Removes VCS part if any, git:http://.. => http://..
+func removeVCSPart(url string) string {
+	parts := strings.SplitN(url, ":", 3)
+	if len(parts) < 3 {
+		// does not look as foo:bar:...
+		return url
+	}
+	for _, r := range parts[0] {
+		if !isASCIILetter(r) {
+			return url
+		}
+	}
+	for _, r := range parts[1] {
+		if !isASCIILetter(r) {
+			return url
+		}
+	}
+	return parts[1] + ":" + parts[2]
+}
+
+// Returns true if given rune is ASCII letter
+func isASCIILetter(r rune) bool {
```

Comment:
Go style suggests the comment should begin with "isASCIILetter ...".

It's also a good practice (set by Rob Pike) to use the verb "reports" rather than the phrase "returns true" for funcs that return a single `bool` type.

So, this should be:

```Go
// isASCIILetter reports if given rune is an ASCII letter.
```